app/controllers/account/appointments_controller.rb

```diff
@@ -34,6 +34,11 @@ def create
       end
     end
 
+    def destroy
+      Appointment.find(params[:id]).destroy
```

Comment:
I think this needs to be scoped to the appointments for the current member to prevent deleting another person's appointments:

```suggestion
      current_member.appointments.find(params[:id]).destroy
```

Reply:
Thanks, can't believe I missed the scoping part. Thanks for the suggestion. I'll make the necessary changes.